cmd/zt_copy_s2smigration_test.go

```diff
@@ -637,6 +637,7 @@ func (s *cmdIntegrationSuite) TestS2SCopyFromContainerToContainerNoPreserveBlobT
 
 //Attempt to copy from a page blob to a block blob
 func (s *cmdIntegrationSuite) TestS2SCopyFromPageToBlockBlob(c *chk.C) {
+	c.Skip("Enable after setting Account to non-HNS")
 	bsu := getBSU()
```

Comment:
Did we change the test storage account to HNS? I recall that accessing account.dfs.core.windows.net from an account without HNS enabled functioned similarly.

Additionally, I feel like it's not a great idea to disable these things in this fashion-- It's easy to forget about.